src/components/Header/MobileHeader.js

```diff
@@ -44,7 +44,7 @@ const styles = {
     backgroundColor: '#FFF',
     textDecoration: 'none',
     display: 'inline-block',
-    height: '50px',
+    height: 50,
```

Comment:
Minor thing: why no `px` here but on the next line, you add it?

Reply:
also noticing the X closer on each tab needs some work. I'm not able to change them via the browser to provide values to set them _to_. We can see about next week

Reply:
Cool, we can work on that now. For some reason, React complained that it was suppose to take a number property and not a string. So I had to change it to `50`. instead of `50px`